packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -32,6 +38,22 @@ export class HttpForwarder
     };
   }
 
+  private updateHostHeaders(baseUrl: string, headers?: IHttpNameValue) {
+    // no headers? do nothing
+    if (!headers) return headers;
```

Comment:
Shouldn't we fill these headers even if there are no headers? My gut says you're doing this mostly because of Typescript, isn't it?

Reply:
Nope, `Host: ` header is not necessary for webserver to process the request. I'd assume that client knows better whether Host: should be there or not. And prism should follow that decision.

Reply:
I think that without the host header the client won't be able to route the request at all as it does not know the destination. It seems indeed to be mandatory for http 1.1

Quoting from [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host)

> A Host header field must be sent in all HTTP/1.1 request messages. A 400 (Bad Request) status code will be sent to any HTTP/1.1 request message that lacks a Host header field or contains more than one.

I'm fine with leaving the code as it is right now, but I think in the future we can modify the `headers` parameters so that it's not optional and has at least the `HOST` header.